zio-http/src/main/scala/zio/http/model/headers/values/AcceptCharset.scala

```diff
@@ -0,0 +1,72 @@
+package zio.http.model.headers.values
+
+sealed trait AcceptCharset {
```

Comment:
Let's move this to a new object `object HeaderValues` inside `zio.http.model`.

Reply:
@jdegoes the `HeaderValues` object will become very large. Keeping the `headers.values` package will allow easier parallelisation of development of the header values, i.e. less conflicts during merge ...

Reply:
How about this:

```scala
sealed trait HeaderValue
object HeaderValue {
  final case class Age(...) extends HeaderValue
  ...
}
```

This is a nice and logical way to organize them even if they share no similarities for now. 

It also let's us say at some point, `final case class Multiple(headerValues: Chunk[HeaderValue])` which we can't say right now.